tests/js/spec/views/userFeedback/index.spec.jsx

```diff
@@ -49,7 +50,12 @@ describe('UserFeedback', function () {
       headers: {Link: pageLinks},
     });
 
-    const wrapper = mountWithTheme(<UserFeedback {...params} />, routerContext);
+    const wrapper = mountWithTheme(
+      <OrganizationContext.Provider value={organization}>
```

Comment:
Hmm, doesn't this get inserted with the routerContext?

Reply:
with RTL we make a bunch of providers, but not with enzyme

Reply:
yup, you're right